xtask/codegen/css.ungram

```diff
@@ -934,9 +935,12 @@ AnyCssPageSelector =
 	| CssBogusSelector
 
 // @page name:first,:blank:first {  }
-// 			 ^^^^^^^^^^ ^^^^^^^^^^^^
+//       ^^^^^^^^^^ ^^^^^^^^^^^^
+// NOTE: The CSS Spec uses `<ident>` for the name, but also explicitly states
+// that the name is case sensitive. For simplicity, `<custom-ident>` gets the
+// exact same behavior.
 CssPageSelector =
-	type: CssIdentifier?
+	type: CssCustomIdentifier?
 	pseudos: CssPageSelectorPseudoList
```

Comment:
This seems to be the only case of the spec using `<ident>` but saying it's case sensitive. Rather than trying to handle a special case, I figure just using the `<custom-ident>` makes sense. They have the exact same syntax specification, the latter is just defined as case-sensitive and not a pre-defined CSS keyword.